### ISSUE TITLE
[nih_plug_iced] Two different versions of baseview if updated

### ISSUE THREAD

**Issue by waynevanson** [robbert-vdh/nih-plug#96]
Hi there!

I am trying to use `nih_plug_iced` and found that I have two different versions of `baseview`, which is not allowing me to build.

`nih_plug_iced <- baseview`
`nih_plug_iced <- ice_baseview <- baseview`

These resolve as two incompatible versions of the `baseview` (I think).

First
https://github.com/RustAudio/baseview.git#f0639b78

Second 
https://github.com/RustAudio/baseview.git?rev=1d9806d5bd92275d0d8142d9c9c90198757b9b25#1d9806d5

Currently seeing if patching in Cargo.toml helps.

Running the following gives me this output:

```shell
cargo update &&
git diff Cargo.lock |
grep -n baseview
```

```diff
 name = "baseview"
-source = "git+https://github.com/RustAudio/baseview.git#c129b12ead4f5ac02126f559ceb8ce43cc982200"
+source = "git+https://github.com/RustAudio/baseview.git#f0639b787bbda506434d3f6b5c91e94ca59904c6"
 name = "iced_baseview"
-source = "git+https://github.com/robbert-vdh/iced_baseview.git?branch=feature/update-baseview#07f1b07d3a8bf2a9af0ce6ee57443105ac99de33"
+source = "git+https://github.com/robbert-vdh/iced_baseview.git?branch=feature/update-baseview#df3a852a15cf0e9fcc8d2b32f5718e56780beaf3"
- "baseview 0.1.0 (git+https://github.com/RustAudio/baseview.git)",
+ "baseview 0.1.0 (git+https://github.com/RustAudio/baseview.git?rev=1d9806d5bd92275d0d8142d9c9c90198757b9b25)",
  "baseview 0.1.0 (git+https://github.com/RustAudio/baseview.git?rev=1d9806d5bd92275d0d8142d9c9c90198757b9b25)",
```

It shows that the newest commit of `iced_baseview` doesn't work as expected.

```diff
-source = "git+https://github.com/robbert-vdh/iced_baseview.git?branch=feature/update-baseview#07f1b07d3a8bf2a9af0ce6ee57443105ac99de33"
+source = "git+https://github.com/robbert-vdh/iced_baseview.git?branch=feature/update-baseview#df3a852a15cf0e9fcc8d2b32f5718e56780beaf3"
```

**Comment by waynevanson** [comment]
Here's a diff from my `Cargo.toml` for those who need a temporary fix.

```diff
-nih_plug = { git = "https://github.com/robbert-vdh/nih-plug.git", features = [
-    "assert_process_allocs",
-    "vst3",
-] }
-nih_plug_iced = { git = "https://github.com/robbert-vdh/nih-plug.git" }
+nih_plug = { git = "https://github.com/waynevanson/nih-plug.git", branch = "fix/iced-baseview-version", features = [
+    "assert_process_allocs",
+    "vst3",
+] }
+nih_plug_iced = { git = "https://github.com/waynevanson/nih-plug.git", branch = "fix/iced-baseview-version" }

```

**Comment by robbert-vdh** [comment]
Have you tried the `vizia-update` branch?

**Comment by waynevanson** [comment]
Closed by #97
